### PR TITLE
Add MinIO supported compression formats.

### PR DESCRIPTION
**Pull request by klauspost**

### CODE REVIEW

**Comment by klauspost** [comment]
Please check formatting. I don't have it set up to test the output.

File: source/reference/minio-cli/minio-mc/mc-sql.rst [hunk 1]
```diff
@@ -230,6 +230,13 @@ Syntax
    - ``BZIP2``
    - ``NONE`` (default)
 
+   Compression schemes supported by MinIO backend only:
```
Comment (ravindk89) [inline source/reference/minio-cli/minio-mc/mc-sql.rst:233]: Syntax is fine - one clarifying question. By `MinIO backend only`, do we mean that these commands only work if `mc sql` is targeting these files stored on MinIO vs another S3 storage?

Comment (klauspost) [reply source/reference/minio-cli/minio-mc/mc-sql.rst:233]: @ravindk89 Files stored on MinIO only.

Comment (ravindk89) [reply source/reference/minio-cli/minio-mc/mc-sql.rst:233]: OK just checking - this is perfect, thank you for opening the pull :)

**Review by ravindk89 — APPROVED** [review]

**Pull request merged**